Kudu.Core/Infrastructure/LockFile.cs

```diff
@@ -287,27 +285,6 @@ private void OnLockReleasedInternal(object sender, FileSystemEventArgs e)
             }
         }
 
-        private static bool IsFileSystemReadOnly()
```

Comment:
Is there a test class for FileSystemHelpers? If so, please add a new unit test for your new method. If a test class doesn't exist, please create one :)